locustfile.py

```diff
@@ -2,11 +2,15 @@
 import os
 import random
 import uuid
-from locust import HttpUser, task, constant
+from locust import HttpUser, task, constant, between
```

Comment:
Prefer the use of `constant_pacing` to try and run a task once every second, injecting as much time as needed per run to make that happen. See https://docs.locust.io/en/stable/writing-a-locustfile.html#wait-time-attribute
```suggestion
from locust import HttpUser, task, constant, constant_pacing
```